pkgs/by-name/li/libblake3/package.nix

```diff
@@ -28,6 +29,20 @@ stdenv.mkDerivation (finalAttrs: {
     tbb_2021_11
   ];
 
+  patches = [
+    (fetchpatch {
+      name = "relax-clang-frontend-compiler-detection.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/BLAKE3-team/BLAKE3/pull/477.patch";
+      hash = "sha256-9qABZTDI3D+m6Tn3I2Zz4qEWEC6pcCZEOpSvLCnbt04=";
+    })
+  ];
+
+  patchPhase = ''
+    pushd ..
+    patchPhase
+    popd
```

Comment:
haha nice :)

Reply:
Yes :)

However, `patch` has builtin support for this, so I suggest replacing it with `patchFlags = [ "-d" ".." ];` so we don't have to mess with phases